libexec/uninstall.ps1

```diff
@@ -0,0 +1,89 @@
+# Uninstall neuro-conda in Windows
+# This script simply automates the official uninstallation instructions:
+# https://docs.anaconda.com/free/anaconda/install/uninstall/#windows
+
+# Set up env variables + functions
+If ($Env:ncDebug){
+    $DebugPreference = "Continue"
+}
+If ($Env:ncCI){
+    Write-Debug "Running inside CI pipeline, turning on non-interactive mode"
+    $Env:ncNoninteractive = $true
+}
+If ($Env:ncNoninteractive){
+    Write-Warning "Running in non-interactive mode - will not prompt for input!"
+}
+If ($Env:username -match " "){
+    $CondaInstallationDirectory = "$Env:public\miniconda3"
+}
+Else {
+    $CondaInstallationDirectory = "$Env:userprofile\miniconda3"
+}
+
+function User-Input {
+    $ans = Read-Host "Press RETURN/ENTER to continue or any other key to abort:"
+    if ($ans -eq ""){
```

Comment:
The logic here is inverse to what the prompt says. RETURN/ENTER exists instead of continuing.

Reply:
Fixed in 25dc36d2a15f3e54da3ed1555c3dc6b3da91c4c4